pkg/cert/legobridge/pki.go

```diff
@@ -217,7 +217,7 @@ func generateCertFromCSR(csrPEM []byte, duration time.Duration, isCA bool) (*x50
 		IsCA:                  isCA,
 		Subject:               csr.Subject,
 		NotBefore:             time.Now(),
-		NotAfter:              time.Now().Add(duration),
+		NotAfter:              time.Now().Add(*duration),
```

Comment:
Dereferencing looks not safe to me. At least such helper methods should use a `time.Duration` instead.

Reply:
Makes sense! Thanks for catching this 🦅 
Fixed with: be6c44cb2344da0a726e02f92c07e6d9f4d44b37